components/mediation/data-publishers/org.wso2.micro.integrator.observability/src/main/java/org/wso2/micro/integrator/observability/metric/handler/prometheus/reporter/PrometheusReporter.java

```diff
@@ -355,5 +365,14 @@ public void initializeServeMetrics() {
     public void initializeArtifactDeploymentMetrics() {
         createMetrics(MetricConstants.SERVICE, MetricConstants.GAUGE, MetricConstants.SERVICE_UP,
                 "Service Status", new String[]{MetricConstants.SERVICE_NAME, MetricConstants.SERVICE_TYPE});
+
+    }
+
+    public void setCounterValue(Counter counter, String serviceName, String serviceType){
+        if(serviceType.equals(SynapseConstants.FAIL_SAFE_MODE_API )) {
```

Comment:
Remove the space after FAIL_SAFE_MODE_API

Reply:
Fixed